doc/user-getting-started.md

```diff
@@ -73,12 +73,29 @@ Options:
                              the host and the enclave
     --host-to-enc-gid-map <host-gid:enc-gid[,host-gid2:enc-gid2,...]>
                          -- comma separated list of gid mappings between
-                             the host and the enclave
+                            the host and the enclave
     --unhandled-syscall-enosys <true/false>
                          -- flag indicating if the app must exit when
                             it encounters an unimplemented syscall
                             'true' implies the syscall would not terminate
                             and instead return ENOSYS.
+    --strace            
+                         -- Use this option to display the strace of the execution
+    --strace-failing
+                         -- When specified, all syscalls that fail will be logged.
+                           Other syscalls will not be logged, unless specified via filter (see below).
+                           Set a breakpoing in _strace_failure_hook to stop execution whenever a
+                           syscall fails. Use breakpoint conditions to control the behavior of the breakpoint.
+    --strace-filter "SYS_name1:SYS_name2:..."
+                         -- Specify the set of syscalls to be traced. When filters are specified, only those
+                           syscalls specified in the filter will be traced, in addition to failing syscalls if
+                           specified as described above.
+                           E.g: To trace open and mprotect syscalls, specify
+                           --strace-filter "SYS_open:SYS_mprotect"
```

Comment:
(Very minor) The indentation seems to miss by 1 space

Reply:
Thanks for pointing it out, I've fixed it for this line and other parts of this menu